TrackletAlgorithm/TrackletProcessor.h

```diff
@@ -520,7 +542,7 @@ TF::seed Seed, // seed layer combination (TC::L1L2, TC::L3L4, etc.)
   uint16_t N // maximum number of steps
 > void
   TrackletProcessor(
-		    const BXType bx,  BXType& bx_o, const LUTTYPE lut[lutsize], const REGIONLUTTYPE regionlut[regionlutsize], const AllStubInnerMemory<InnerRegion> innerStubs[NASMemInner], const AllStubMemory<OuterRegion>* outerStubs, const VMStubTEOuterMemoryCM<OuterRegion,RZBins,PhiBins,NTEUnits>* outerVMStubs, TrackletParameterMemory * const trackletParameters, TrackletProjectionMemory<BARRELPS> projout_barrel_ps[TC::N_PROJOUT_BARRELPS], TrackletProjectionMemory<BARREL2S> projout_barrel_2s[TC::N_PROJOUT_BARREL2S], TrackletProjectionMemory<DISK> projout_disk[TC::N_PROJOUT_DISK])
+		    const BXType bx,  BXType& bx_o, const int lut[], const int regionlut[], const AllStubInnerMemory<InnerRegion> innerStubs[NASMemInner], const AllStubMemory<OuterRegion>* outerStubs, const VMStubTEOuterMemoryCM<OuterRegion,RZBins,PhiBins,NTEUnits>* outerVMStubs, TrackletParameterMemory * const trackletParameters, TrackletProjectionMemory<BARRELPS> projout_barrel_ps[TC::N_PROJOUT_BARRELPS], TrackletProjectionMemory<BARREL2S> projout_barrel_2s[TC::N_PROJOUT_BARREL2S], TrackletProjectionMemory<DISK> projout_disk[TC::N_PROJOUT_DISK])
```

Comment:
@jasonfan393 Does changing the types of the arrays from `LUTTYPE` and `REGIONLUTTYPE` to `int` change the resource utilization at all?

Reply:
changed this to int just because I was copying the style of the VMR

just tried it out with changing all the ints to ap_uints and putting the LUTTYPE back in. Resource usage seems to be identical (at least at the synthesis estimate).  Have you observed this making a difference elsewhere? I can change it back to be safe

Reply:
@jasonfan393 Sorry it took so long to follow up on this.

If the resource usage is identical, then I don't think I care either way. If you do go with the version using `int`, just be sure to remove the definitions of `LUTTYPE`, `lutsize`, `REGIONLUTTYPE`, and `regionlutsize`, since they aren't used anywhere in that case.

Reply:
I've just left the LUTTYPE , lutsize etc. in for now